sklearn/feature_selection/_sequential.py

```diff
@@ -57,6 +57,10 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin, BaseEstimator
     tol : float, default=None
         If the score is not incremented by at least `tol` between two
         consecutive feature additions or removals, stop adding or removing.
+
+        `tol` can be negative when removing features. It can be useful to reduce
+        the number of features at the cost of a small decrease in the score.
```

Comment:
I think it's good to directly say that removing features happen during backward selection.

```suggestion
        `tol` can be negative when removing features using `direction="backward"`.
        It can be useful to reduce the number of features at the cost of a
        small decrease in the score.
```